docs/guides/rabbitmq/quickstart/yamls/quickstart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kubedb.com/v1alpha2
+kind: RabbitMQ
+metadata:
```

Comment:
Remove this yamls folder.  Refer the yamls from examples folder in the md files.